src/index.ts

```diff
@@ -307,6 +307,8 @@ export class Probot {
 
   public start() {
     this.log.info(`Running Probot v${this.version}`);
+    this.log.info(`NodeJS environment: ${process.env.NODE_ENV}`);
+    this.log.info(`Log level: ${this.log.level}`);
```

Comment:
`NODE_ENV` can be undefined. As mentioned in the other issue, I'd like to remove accessing environment variables from within the `probot` instance. Instead we should probably add an `env` option to the `Probot` constructor, if we need that at all. I'm currently unsure where we use the `NODE_ENV` variable, I'd have to doble check.

`this.log` is a [pino](https://getpino.io/#/) instance, so you can pass extra information as the first argument, e.g. 

```js
    this.log.info({env: process.env.NODE_ENV || "development", logLevel: this.log.level}, `Running Probot v${this.version}`);
```

Thinking more about these logs, I think they probably don't belong here. The rather belong in the `probot` binary code, or `Probot.run`

Reply:
> Thinking more about these logs, I think they probably don't belong here. The rather belong in the probot binary code, or Probot.run

Understood, I'll update the patch then.

PS Can you please elaborate on `probot binary code`? Isn't `Probot.run` part of it?